sdk/template/Azure.Template/tests/Samples/Sample1_HelloWorldAsync.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+using Azure.Template.Models;
+using NUnit.Framework;
+
+namespace Azure.Template.Tests.Samples
```

Comment:
It looks the file content has nothing related to 'HelloWorld', can we rename the file name to 'TemplateSamples' directly? (same as the class name)

Reply:
No.  each operation will have two sample file (sync, and async), and the two files will define the same class. The file name is consistent with the guideline. Please check https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-samples